src/components/blockquote.tsx

```diff
@@ -16,31 +17,37 @@ interface Props {
     format: Format;
 }
 
-const styles = (format: Format): SerializedStyles => css`
-    font-style: italic;
-    position: relative;
-    margin: ${remSpace[4]} 0 ${remSpace[9]} 0;
-    padding: 0 ${remSpace[6]};
+const styles = (format: Format): SerializedStyles => {
+    const { kicker } = getPillarStyles(format.pillar);
+    return css`
+        font-style: italic;
+        position: relative;
+        margin: ${remSpace[4]} 0 ${remSpace[9]} 0;
+        padding: 0 ${remSpace[5]};
+
+        svg {
+                height: 62.4px;
+                margin-left: -9.2625px;
+                margin-bottom: -12px;
+                margin-top: 4.4px;
```

Comment:
Regarding the use of pixel values here, what happens if you scale the page up and down (`cmd +` and `cmd -` in a browser)?